actions/setup_e2e_tests.sh

```diff
@@ -55,7 +55,7 @@ st2ctl reload --register-all
 
 # Robotframework requirements
 cd st2tests
-sudo pip install --upgrade pip
+sudo pip install --upgrade "pip>=9.0,<9.1"
 sudo pip install --upgrade virtualenv
```

Comment:
Need to pin virtualenv as well here (to `15.1.0` which ships with pip 9.0)

Reply:
And same things, do `virtualenv --no-download` 2 lines below as well, otherwise it will be using latest pip 10.

Reply:
Good catch - pushed those changes.